CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+Contributing to Opencast
+========================
+
+This is a short outline of what to pay attention to when contributing to Opencast. If you are interested in a long
+document defining rules and recommendations as well as a detailed explanation of Opencast's branching model, take a look
+at [the development process documentation](https://docs.opencast.org/develop/developer/development-process/). But this
+guide should suffice for most contributions.
+
+
+Issue Tracker
+-------------
+
+Opencast uses [Jira](https://opencast.jira.com) for tracking issues. Each pull request should be accompanied by a ticket
+in Jira. The issue identifier should also be used in the title of the pull request and the commits. E.g.: `MH-12345,
```

Comment:
@lkiesow The comma after the issue number seems somewhat superfluous. What do you think about "MH-12345 Description" instead of "MH-12345, Description"?

Reply:
This is the exact example from the development process docs. Also, to be honest, I like to have a separator between the ticket identifier and the summary. These are two different parts after all. But then, this is also just an example. If you like me to, I can remove it though, to just leave the choice to the contributor.

Reply:
No strong opinion from my side, just wanted to mention it.

Reply:
I've simply removed the suggestion. There should be enough real-world examples to follow.